exporter/splunkhecexporter/README.md

```diff
@@ -66,6 +66,7 @@ The following configuration options can also be configured:
 - `telemetry/enabled` (default: false): Specifies whether to enable telemetry inside splunk hec exporter.
 - `telemetry/override_metrics_names` (default: empty map): Specifies the metrics name to overrides in splunk hec exporter.
 - `telemetry/extra_attributes` (default: empty map): Specifies the extra metrics attributes in splunk hec exporter.
+- `batcher`: Specifies batching configuration on the exporter. Information about the configuration can be found [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
```

Comment:
Adding a link here is super helpful, but the link doesn't seem to include any information for the [actual config option](https://github.com/open-telemetry/opentelemetry-collector/blob/7b046d9aaf94e92b2bfc6c25bec4ac33ba5082f2/exporter/exporterbatcher/config.go#L15). I don't actually see any reference outside of code, so I don't have a good answer for what to have here.

Also, the comment above the config gives me a bit of hesitation with adopting the exporter batcher at this point. @dmitryax, do you think it's worth moving forward now, or should we wait to adopt it? Especially considering the Splunk HEC exporter is considered beta.

Reply:
yeah the reference is just a placeholder when the documentation eventually surfaces. By default the batcher config is disabled so it shouldn't affect existing users. We can:
- remove this from README reference if we don't want users to try it or
- add experimental tags in the config

Having batching enabled in splunk hec exporter is in our short term roadmap so we are hoping there is at least an option to enable it.

Reply:
That all makes sense, thanks for giving more information here! I'll defer to the code owners then. 👍

Reply:
Let's put an experimental warning here 👍

Reply:
I updated the documentations around. Let me know what you think @crobert-1 @dmitryax